### DISCUSSION TITLE
Alldebrid api changes impact ?

### DISCUSSION THREAD

**Discussion by 666dsa666** [rogerfar/rdt-client#627] [Q&A]
Would anyone be able to confirm the impact of recent changes made by alldebrid ?

[This announcement](https://i.redd.it/dp1zfurox23e1.jpeg)

My premium account will get renewed tomorrow, I was planning on setting up RDT this weekend but if needed I'll switch to premiumize which seems to be having less issues than RD and AD.

Thanks !

**Comment by rogerfar** [comment]
From that email I'm not sure which feature they're disabling? It almost feels like they're shutting down their api?

**Reply by 666dsa666** [reply]
Yes I believe it's their API too, all third party apps would be disabled by their words.

**Comment by rogerfar** [comment]
So far it seems all OK? The only thing that was removed is the ability to find out if a certain torrent is already cached.
